### PR TITLE
Fix overlapping bars in single group

### LINKED ISSUE #29335: [@fluentui/react-charting] Grouped Vertical Bar Chart - Bar overlaps when in single series
Hi team,

I noticed that the bar overlaps when I only have a single series. Any idea how I can resolve this issue? Thanks.

@fluentui/react-charting v5.17.7

Single Series:
image

Multi Series
image

Edited:
I see that the width is being set to a negative value based on console log error.
image

_Originally posted by @jekcalingan in https://github.com/microsoft/fluentui/discussions/29326_

### PR DESCRIPTION
**Pull request by krkshitij**
## Previous Behavior

Before

## New Behavior

After

## Related Issue(s)

- Fixes #29335

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/d3ea7d93b6d01f3355651d9635e3f0197dc208ea'>d3ea7d93b6d01f3355651d9635e3f0197dc208ea</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-charting/src/components/GroupedVerticalBarChart/GroupedVerticalBarChart.base.tsx [hunk 1]
```diff
@@ -388,7 +391,10 @@ export class GroupedVerticalBarChartBase extends React.Component<
       xAxisElement && tooltipOfXAxislabels(tooltipProps);
     }
     return (
-      <g key={singleSet.indexNum} transform={`translate(${xScale0(singleSet.xAxisPoint)}, 0)`}>
+      <g
+        key={singleSet.indexNum}
+        transform={`translate(${xScale0(singleSet.xAxisPoint) + (xScale0.bandwidth() - this._groupWidth) / 2}, 0)`}
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/GroupedVerticalBarChart/GroupedVerticalBarChart.base.tsx:396]: do we need to handle rtl here?

**Review by AtishayMsft — APPROVED** [review]

File: packages/react-charting/src/components/GroupedVerticalBarChart/GroupedVerticalBarChart.base.tsx (hunk 1 above)
Comment (krkshitij) [reply packages/react-charting/src/components/GroupedVerticalBarChart/GroupedVerticalBarChart.base.tsx:396]: It works fine in RTL as well

Screenshot 2023-10-05 110253

**Pull request merged**